src/main/java/mate/academy/dao/jdbc/ProductDaoJdbcImpl.java

```diff
@@ -0,0 +1,108 @@
+package mate.academy.dao.jdbc;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.academy.dao.ProductDao;
+import mate.academy.exceptions.DataProcessingException;
+import mate.academy.lb.Dao;
+import mate.academy.model.Product;
+import mate.academy.util.ConnectionUtil;
+
+@Dao
+public class ProductDaoJdbcImpl implements ProductDao {
+
+    @Override
+    public Product create(Product element) {
+        String query = "INSERT INTO products (productname, productprice) VALUE (?, ?);";
+        try (Connection connection = ConnectionUtil.getConnection()) {
+            PreparedStatement statement = connection
+                    .prepareStatement(query, Statement.RETURN_GENERATED_KEYS);
+            statement.setString(1, element.getName());
+            statement.setDouble(2, element.getPrice().doubleValue());
+            statement.executeUpdate();
+            ResultSet resultSet = statement.getGeneratedKeys();
+            if (resultSet.next()) {
+                element.setId(resultSet.getLong(1));
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Data processing exception", e);
```

Comment:
let's add more information about the product and about the problem which we have to all catch blocks